### PR TITLE
fixing the example apps

### PR DESCRIPTION
**Pull request by kartikeyakirar**
this fixes : https://github.com/insightsengineering/teal.widgets/issues/215

### CODE REVIEW

File: R/optionalInput.R [hunk 1]
```diff
@@ -52,101 +52,99 @@
...
+#'   })
+#'   output$c4_out <- renderPrint({
+#'     input$c4
+#'   })
+#'   output$c5_out <- renderPrint({
+#'     input$c5
+#'   })
+#'   output$c6_out <- renderPrint({
+#'     input$c6
+#'   })
```
Comment (averissimo) [inline R/optionalInput.R:143]: How about compacting this a bit?

```suggestion
#'   output$c1_out <- renderPrint(input$c1)
#'   output$c2_out <- renderPrint(input$c2)
#'   output$c3_out <- renderPrint(input$c3)
#'   output$c4_out <- renderPrint(input$c4)
#'   output$c5_out <- renderPrint(input$c5)
#'   output$c6_out <- renderPrint(input$c6)
```

**Review by averissimo — CHANGES_REQUESTED** [review]
Can you also change the remaining `shinyApp(ui$...` call?

- `plot_with_settings_srv`


Otherwise it LGTM. A small comment about some code that can be compacted, but nothing else

`checks` also run without problems locally

- All examples run manually and via `devtools::run_examples(fresh = T)`
- Vignette examples also run without a problem

**Comment by kartikeyakirar** [comment]
> Can you also change the remaining shinyApp(ui$... call?

thanks @averissimo, I missed that example. updated here [798aa37](https://github.com/insightsengineering/teal.widgets/pull/219/commits/798aa375d01bc16fae7734dd714a224ddc237dda)

**Review by averissimo — APPROVED** [review]
All good

- Tested all examples manually and via `devtools::{run_examples,check}` and they work fine
- No `shiny::` prefix in examples

**Pull request merged**